src/Psalm/Internal/PhpVisitor/Reflector/ClassLikeNodeScanner.php

```diff
@@ -1580,9 +1580,8 @@ private function visitPropertyDeclaration(
                 }
 
                 if ($doc_var_group_type) {
-                    $property_storage->type = count($stmt->props) === 1
-                        ? $doc_var_group_type
-                        : $doc_var_group_type;
+                    // TODO: What was up with this ternary?
```

Comment:
Probably a leftover from the `clone` removal refactoring, it's OK to remove this comment :)